src/components/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+import { FilterBy } from '../types/FilterEnum';
+
+interface Props {
+  todos: Todo[];
+  visibleTodos: Todo[];
+  filterBy:FilterBy;
+  setFilterBy:React.Dispatch<React.SetStateAction<FilterBy>>;
+}
+
+export const Footer: FC<Props> = ({
+  todos,
+  visibleTodos,
+  filterBy,
+  setFilterBy,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${todos.length} items left`}
```

Comment:
Use visibleTodos.length instead of todos.length to show the count of items left.